WGLMakie/test/picking.jl

```diff
@@ -0,0 +1,195 @@
+@testset "picking" begin
+    scene = Scene(size = (130, 370))
+    campixel!(scene)
+    
+    sc1 = scatter!(scene, [20, NaN, 20], [20, NaN, 50], marker = Rect, markersize = 20)
+    sc2 = scatter!(scene, [50, 50, 20, 50], [20, 50, 80, 80], marker = Circle, markersize = 20, color = [:red, :red, :transparent, :red])
+    ms = meshscatter!(scene, [20, NaN, 50], [110, NaN, 110], markersize = 10)
+    l1 = lines!(scene, [20, 50, 50, 20, 20], [140, 140, 170, 170, 140], linewidth = 10)
+    l2 = lines!(scene, [20, 50, NaN, 20, 50], [200, 200, NaN, 230, 230], linewidth = 20, linecap = :round)
+    ls = linesegments!(scene, [20, 50, NaN, NaN, 20, 50], [260, 260, NaN, NaN, 290, 290], linewidth = 20, linecap = :square)
+    tp = text!(scene, Point2f[(15, 320), (NaN, NaN), (15, 350)], text = ["█ ●", "hi", "●"], fontsize = 20, align = (:left, :center))
+    t = tp.plots[1]
+
+    i = image!(scene, 80..110, 20..50, rand(RGBf, 2, 2))
+    s = surface!(scene, 80..110, 80..110, rand(2, 2))
+    hm = heatmap!(scene, [80, 110], [140, 170], [1 2; 3 4])
+    # mesh coloring should match triangle placements
+    m = mesh!(scene, Point2f.([80, 80, 110, 110], [200, 230, 200, 230]), [1 2 3; 2 3 4], color = [1,1,1,2])
+    vx = voxels!(scene, [65, 125], [245, 305], [-1, 1], reshape([1,2,3,4], (2,2,1)), shading = NoShading)
+    vol = volume!(scene, 80..110, 320..350, -1..1, rand(2,2,2))
+
+    # render one frame to generate picking texture
+    colorbuffer(scene);
```

Comment:
Ah, so this was the trick for rendering a frame in CI.

Reply:
It might also depend on where the file is included. I (now) have it after the refimg tests since those need to work the same for every backend, and it seems to work